README.md

```diff
@@ -138,6 +138,10 @@ our company and culture evolve, so should this document.
 repeating the mistakes of the past.
 - Ensure you challenge designs that you see problems with instead of implementing them without questioning.
 
+#### Spread knowledge
+- Create documentation for the software you write (sometimes [even before you write it](http://tom.preston-werner.com/2010/08/23/readme-driven-development.html))
```

Comment:
I guess my reasoning for "Always Grow" was from the perspective of: "It's easy to defer tasks to people who already know how to do those tasks. Rather then letting things be done over and over by the same small group, pair with people who are having issues and enable them to solve those problems on their own"

But "Communication" also sounds appropriate. I guess it feels like were missing the motivation of MVD. Could include something like "Documentation should enable developers to get setup and running with no external help". Currently it just says "Have MVD - it's important!"